lib/mutant.rb

```diff
@@ -41,6 +41,7 @@ module Mutant
   SCOPE_OPERATOR = '::'
 end # Mutant
 
+require 'mutant/transform'
```

Comment:
The fact this has to move up the require chain to toplevel, indicates it likely belongs to a separate library.

Once I make nicer "input error location rendering" this is worthy of a separate lib.